tests/scanners/test.javascript.js

```diff
@@ -1,7 +1,10 @@
 import ESLint from 'eslint';
 
-import { DEPRECATED_APIS, ESLINT_ERROR, ESLINT_RULE_MAPPING, TEMPORARY_APIS,
-         VALIDATION_ERROR, VALIDATION_WARNING } from 'const';
+import {
+  DEPRECATED_APIS, ESLINT_ERROR, ESLINT_RULE_MAPPING, TEMPORARY_APIS,
```

Comment:
Might be more manageable to make these one per line and then scanning + sorting is easier.